### PR TITLE
Implement adjustStreamVolume in Android

### PR DESCRIPTION
**Pull request by ghsdh3409**
* Implement adjustStreamVolume in Android
* It closes https://github.com/hayanmind/mimiking-rn/issues/826

### CODE REVIEW

File: android/src/main/java/com/zmxv/RNSound/RNSoundModule.java [hunk 1]
```diff
@@ -400,6 +400,12 @@ public void removeRouteChangeListener() {
     }
   }
 
+  @ReactMethod
+  public void adjustStreamVolume(final int streamType, final int direction, final int flags) {
```
Comment (ohtangza) [inline android/src/main/java/com/zmxv/RNSound/RNSoundModule.java:404]: Should the parameters be final?

File: index.d.ts [hunk 2]
```diff
@@ -15,6 +15,16 @@ export default class Sound {
   static CACHES: string
 
   /**
+   * (Android Only) Adjusts stream volum.
+   * Parameter: Please refer constants in https://developer.android.com/reference/android/media/AudioManager.html
+   *
+   * @param streamType The stream type to adjust. Refer constants in https://developer.android.com/reference/android/media/AudioManager.html
```
Comment (ohtangza) [inline index.d.ts]: Amazing...!

File: index.d.ts [hunk 3]
```diff
@@ -15,6 +15,16 @@ export default class Sound {
   static CACHES: string
 
   /**
+   * (Android Only) Adjusts stream volum.
```
Comment (ohtangza) [inline index.d.ts]: "volum" --> "volume".

**Review by ohtangza — APPROVED** [review]
I have minor comments. You can fix it as you wish!

File: android/src/main/java/com/zmxv/RNSound/RNSoundModule.java (hunk 1 above)
Comment (ghsdh3409) [reply android/src/main/java/com/zmxv/RNSound/RNSoundModule.java:404]: I just followed other methods in the same file. But, It is not needed to be final, when I check official document.

**Pull request merged**